services/api/src/resources/project/harborSetup.ts

```diff
@@ -226,8 +226,8 @@ async function resetHarborWebhook(sqlClient: MariaClient, harborClient, lagoonPr
           }
         ],
         event_types: [
```

Comment:
my one question here is whether we need to wrap this in an API check to make it compatible with Harbor v1?

Reply:
Good catch. I'll write that in

Reply:
![giphy-downsized](https://user-images.githubusercontent.com/4373945/119783155-4bfa2a00-bf10-11eb-8a54-9cb1b9d958bd.gif)